test/functional/test_framework/test_node.py

```diff
@@ -113,6 +113,8 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, timeout_factor, bitc
 
         if self.version_is_at_least(190000):
             self.args.append("-logthreadnames")
+        if self.version_is_at_least(200000):
```

Comment:
v0.20.0 is already released and obviously doesn't include this new feature that is not even merged on master.
```suggestion
        if self.version_is_at_least(209900):
```

Reply:
Oh, thanks! Fix pushed.